src/components/Inputs/HiddenField/HiddenField.tsx

```diff
@@ -1,15 +1,17 @@
 import type TreeNode from "@/types/TreeNode";
+import { IsString } from "@/types/TypeGuards";
 
 export interface HiddenFieldProps {
   data: TreeNode;
-  defaultValue?: unknown;
+  value?: unknown;
 }
 
-const HiddenField = ({ defaultValue, data }: HiddenFieldProps) => {
-  const { name, attributes } = data;
-  const { hiddenValue } = attributes;
+const HiddenField = ({ data, value }: HiddenFieldProps) => {
+  const {
+    attributes: { name },
+  } = data;
 
-  return <input type="hidden" name={name} value={hiddenValue} defaultValue={defaultValue as string} />;
+  return <input type="hidden" name={name} value={IsString(value) ? value : ""} />;
```

Comment:
Ici tu pourrais faire ça je pense

```suggestion
  return <TextField type="hidden" name={name} value={value}  />;
```